packages/http-server-js/vitest.config.ts

```diff
@@ -1,4 +1,11 @@
 import { defineConfig, mergeConfig } from "vitest/config";
 import { defaultTypeSpecVitestConfig } from "../../vitest.workspace.js";
 
-export default mergeConfig(defaultTypeSpecVitestConfig, defineConfig({}));
+export default mergeConfig(
+  defaultTypeSpecVitestConfig,
+  defineConfig({
```

Comment:
were some test getting picked up here without this?

Reply:
The e2e tests were being picked up without this here. Since those require an extra build step (to emit the generated files) I didn't want to treat those separately (as part of test:e2e)